app/pages/notifications/index.jsx

```diff
@@ -1,12 +1,20 @@
-import PropTypes from 'prop-types';
-import React from 'react';
 import counterpart from 'counterpart';
-import Translate from 'react-translate-component';
-import talkClient from 'panoptes-client/lib/talk-client';
+import {
+  arrayOf,
+  bool,
+  func,
+  oneOfType,
+  shape,
+  string
+} from 'prop-types';
+import React, { useEffect, useState } from 'react';
```

Comment:
The `React` global isn't needed to parse JSX any more. Importing it won't break anything but it doesn't do anything either. 

```suggestion
import { useEffect, useState } from 'react';
```

Reply:
👍 
I updated `eslintrc` so the linter wouldn't complain with noted change.